src/legacy/core_plugins/interpreter/init.js

```diff
@@ -47,8 +50,9 @@ export default async function (server /*options*/) {
     };
```

Comment:
Not related to this PR, but does the interpreter need to handle injecting ui app vars into `canvas`?  Especially the reporting config variable ... this seems like it doesn't belong in the interpreter code.